### ISSUE TITLE
firefox 49.0.2 titlebar/menubar issue

### ISSUE THREAD

**Issue by cwriordan** [Javier-DarthPalpatine/firefox-hide-caption-titlebar-plus#334]
firefox version: 49.0.2.
addon version: 3.1.5

when checking box "system titlebar always hidden", the menu bar is hidden instead of the title bar. did not occur in firefox 49.0.1.

**Comment by Gitiztheshit** [comment]
YEP!!! Your g*dd*mned "addon" broke Firefox once again! Get your s*it together!!!

**Comment by Javier-DarthPalpatine** [comment]
This is in MS Windows right?

FYI: I'm using hctp in Fx 49.0.2 in Win 7 64.   Tested also in Win10
